data_raw/rcd.R

```diff
@@ -5,7 +5,7 @@ releases <- c(
   # "2022_01_28" = "7f2200332aab693d2ef4149a0f67b7708a2db516",
   # "2022_02_28" = "fc3620365f38c36e45880cbdebfe16508d902fa6",
   # nolint end
-  "2022_06_27" = "c1a972e7a177eb4556aa01e6a54d27d6b4d32782"
+  "2022_06_27" = "33ab12d00478c72b74d82150dbbd748af196dccf"
```

Comment:
is this the right commit? i checked the logs of the RCD/main for the issue and it has a different commit in the log (please let me know if i'm looking at the log wrong).
<img width="764" alt="Screen Shot 2022-09-08 at 5 19 24 PM" src="https://user-images.githubusercontent.com/39424598/189247642-a5eaf6b1-52f4-493d-96ed-3bb12f467690.png">

Reply:
this is my only comment! the rest looks good!